.gitignore

```diff
@@ -1 +1,2 @@
-.DS_Store
\ No newline at end of file
+.DS_Store
+.idea
```

Comment:
I understand what gitignore is doing and why it now contains .idea from your webstorm text editor. Sweet!